src/core/model_utils.py

```diff
@@ -358,3 +360,59 @@ def is_svg(f):
     except et.ParseError:
         pass
     return tag == '{http://www.w3.org/2000/svg}svg'
+
+
+class AbstractLastModifiedModelQuerySet(models.query.QuerySet):
+    def update(self, *args, **kwargs):
+        kwargs['last_modified'] = timezone.now()
+        super().update(*args, **kwargs)
+
+
+class AbstractLastModifiedModelManager(models.Manager):
+    def get_queryset(self):
+        # this is to use your custom queryset methods
+        return AbstractLastModifiedModelQuerySet(self.model, using=self._db)
+
+
+class AbstractLastModifiedModel(models.Model):
+    last_modified = models.DateTimeField(
+        auto_now=True,
+        auto_now_add=False,
```

Comment:
We should drop this as well, as want the date_modified to be populated on creation too.